### ISSUE TITLE
Quest: Deathstalkers in Shadowfang

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#84]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Shadowfang Keep / NPC: Deathstalker Vincent / Quest: Deathstalkers in Shadowfang
Link: https://www.wowhead.com/wotlk/quest=1098/deathstalkers-in-shadowfang

Description: playback ok.

Completion: Despite the voice-over is perfect, the text is a description of actions. The narrator voice is more suitable in this case, instead of the male undead voice.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
